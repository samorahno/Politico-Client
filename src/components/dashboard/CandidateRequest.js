import React from 'react';
import '../../styles/dashboard.css';
import Header from '../nav/DashboardHeader';
import Footer from '../nav/DashboardFooter';
import SideBar from '../nav/DashboardSidebar';

const CandidateRequest = () => {
    return (
        <div>
            <Header />
            <SideBar />
            <div className="main-content">
                <div className="title">
					&nbsp;
                </div>

                <div className="main">

                    <div className="widget widget_table" style={{ overflow: 'auto' }}>
                        <div className="title heado">Candidate Request List</div>

                        <div className="widget_stat widget_stat_table">
                            <div style={{ overflow: 'auto' }}>
                                <table style={{ overflow: 'scroll' }}>
                                    <thead >
                                        <tr>
                                            <th>S/N</th>
                                            <th>Candidate</th>
                                            <th>Office</th>
                                            <th>Date Created</th>
                                            <th>Action</th>
                                        </tr>
                                    </thead>

                                    <tbody id="interestContainer">
                                        <tr>
                                            <td>1</td>
                                            <td>A</td>
                                            <td>B</td>
                                            <td>C</td>
                                            <td>D</td>
                                        </tr>
                                        <tr>
                                            <td>2</td>
                                            <td>A</td>
                                            <td>B</td>
                                            <td>C</td>
                                            <td>D</td>
                                        </tr>
                                    </tbody>

                                </table>
                            </div>
                        </div>
                    </div>

                </div>
            </div>
            <Footer />
        </div>

    );
};

export default CandidateRequest;
