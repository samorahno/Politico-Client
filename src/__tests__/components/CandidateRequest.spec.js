import React from 'react';
import { shallow } from 'enzyme';
import CandidateRequest from '../../components/dashboard/CandidateRequest';

describe('Candidate Request Page', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<CandidateRequest />);
        expect(component).toMatchSnapshot();
    });
});
