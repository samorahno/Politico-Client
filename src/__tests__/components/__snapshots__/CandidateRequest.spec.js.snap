// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Candidate Request Page should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
